src/linear_algebra/matrix/nonsingular_inverse.lean

```diff
@@ -36,11 +57,111 @@ open equiv equiv.perm finset
 
 variables (A : matrix n n α) (B : matrix n n α)
 
+/-! ### Matrices are `invertible` iff their determinants are -/
+
+section invertible
+
+/-- A copy of `inv_of_mul_self` using `⬝` not `*`. -/
+@[simp] protected lemma inv_of_mul_self [invertible A] : ⅟A ⬝ A = 1 := inv_of_mul_self A
+
+/-- A copy of `mul_inv_of_self` using `⬝` not `*`. -/
+@[simp] protected lemma mul_inv_of_self [invertible A] : A ⬝ ⅟A = 1 := mul_inv_of_self A
```

Comment:
@jcommelin, the linter doesn't like these as simp lemmas as (at least for now) `⅟` is not simp-normal.

```suggestion
protected lemma inv_of_mul_self [invertible A] : ⅟A ⬝ A = 1 := inv_of_mul_self A

/-- A copy of `mul_inv_of_self` using `⬝` not `*`. -/
protected lemma mul_inv_of_self [invertible A] : A ⬝ ⅟A = 1 := mul_inv_of_self A
```